ftl/node/main.py

```diff
@@ -46,9 +46,14 @@
     action='store',
     help='The path where the application data sits.')
 
+parser.add_argument("-v", "--verbose",
+    const=1, default=0, type=int, nargs="?",
+    help="increase verbosity: 0 = only warnings, 1 = info, 2 = debug. " +
```

Comment:
I kind of prefer using the actual strings (debug, warning, info) etc. as flags. WDYT?

Reply:
sgtm